crates/connectors/ndc-postgres/src/schema.rs

```diff
@@ -626,8 +626,14 @@ fn map_type_representation(
         metadata::TypeRepresentation::Float64 => models::TypeRepresentation::Float64,
         metadata::TypeRepresentation::Int16 => models::TypeRepresentation::Int16,
         metadata::TypeRepresentation::Int32 => models::TypeRepresentation::Int32,
-        metadata::TypeRepresentation::Int64 => models::TypeRepresentation::Int64,
-        metadata::TypeRepresentation::BigDecimal => models::TypeRepresentation::BigDecimal,
+        // Int64 returns a number.
+        metadata::TypeRepresentation::Int64 => models::TypeRepresentation::JSON,
+        // Int64AsString returns a string.
+        metadata::TypeRepresentation::Int64AsString => models::TypeRepresentation::Int64,
+        // BigDecimal returns a number.
+        metadata::TypeRepresentation::BigDecimal => models::TypeRepresentation::JSON,
+        // BigDecimalAsString returns a string.
+        metadata::TypeRepresentation::BigDecimalAsString => models::TypeRepresentation::BigDecimal,
```

Comment:
This is the mapping between our internal type representations and ndc-spec type representations.